pkg/ami/auto_resolver.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/weaveworks/eksctl/pkg/utils"
 )
 
+const (
+	// OwnerIDUbuntu1804Family is the owner ID used for Ubuntu AMIs
+	OwnerIDUbuntu1804Family = "099720109477"
+
+	// OwnerIDWindowsFamily is the owner ID used for Ubuntu AMIs
+	OwnerIDWindowsFamily = "801119661308"
```

Comment:
I think we should leave these unexported as they aren't (and shouldn't be) used outside of this package.